test/mockClient.js

```diff
@@ -0,0 +1,40 @@
+const walletResponses = require('./data/walletResponses.json');
+
+const {
+  getWalletResponse,
+  getMultisigResponse,
+  getWalletInfoResponse,
+  getMSWalletInfoResponse,
+  createMSWalletResponse,
+  createWalletResponse,
+  getAccountsResponse,
+  getAccountInfoResponse,
+  getAccountBalanceResponse,
+  getAccountHistoryResponse,
+  createAccountResponse,
+  getProposalMTXResponse,
+} = walletResponses;
+
+
+const getClientMock = () => ({
```

Comment:
Nice. I wonder if there is a way that would make sense to generalize this and put this out as a testing utility.

Reply:
To live in `bpanel-utils`?

Reply:
Yeah, possibly. Maybe even have a separate library just for testing. Not really sure, but I could see this being useful in lots of situations, especially since the bclient API is going to be constant throughout.

Reply:
I think it could make sense to go in `bpanel-utils` eventually